client/driver/docker.go

```diff
@@ -947,3 +961,51 @@ func (h *DockerHandle) run() {
 		}
 	}
 }
+
+// collectStats starts collecting resource usage stats of a docker container
+func (h *DockerHandle) collectStats() {
+	statsCh := make(chan *docker.Stats)
+	statsOpts := docker.StatsOptions{ID: h.containerID, Done: h.doneMonitoring, Stats: statsCh, Stream: true}
```

Comment:
@dadgar `doneCh` is currently `chan struct{}`, so I needed another channel, but changing the type of `doneCh` so that we can use with `Stats`